korgan/legal_calc.py

```
@@ -145,16 +196,114 @@ def claimant_is_individual(case_context: str) -> bool | None:
+    all_text = f"{case_context}\n{text}".lower()
+
+    if any(marker in all_text for marker in _AMBIGUOUS_CLASSIFICATION_MARKERS):
+        return NEEDS_CALCULATION_MARKER
+    if "моральн" in all_text and not all(word in all_text for word in ("чест", "достоин")):
+        return NEEDS_CALCULATION_MARKER
+    if _potential_identity_exemption(case_context):
+        return NEEDS_CALCULATION_MARKER
+
+    exemption = _explicit_exemption_reason(case_context, text)
+    if exemption:
+        return f"0 тенге (освобождение от уплаты: {exemption}; статья 668 Налогового кодекса РК)"
```

Ruff (0.16.1)

[warning] String contains ambiguous `Р` (CYRILLIC CAPITAL LETTER ER). Did you mean `P` (LATIN CAPITAL LETTER P)?

(RUF001)